lib/src/utils.dart

```diff
@@ -4,21 +4,32 @@
 // license that can be found in the LICENSE file or at
 // https://opensource.org/licenses/MIT.
 
-import 'package:path/path.dart' as p;
+import 'dart:io';
+
 import 'package:source_span/source_span.dart';
 
 // The sass package's API is not necessarily stable. It is being imported with
 // the Sass team's explicit knowledge and approval. See
 // https://github.com/sass/dart-sass/issues/236.
+import 'package:sass/src/ast/sass.dart';
 import 'package:sass/src/ast/node.dart';
-import 'package:sass/src/importer/utils.dart' show resolveImportPath;
+import 'package:sass/src/importer/filesystem.dart';
 export 'package:sass/src/utils.dart' show normalizedMap, normalizedSet;
 
 import 'patch.dart';
 
-/// Returns the canonical version of [path].
-String canonicalizePath(String path) {
-  return p.canonicalize(resolveImportPath(path));
+/// Returns the canonical version of [url].
+Uri canonicalize(Uri url) {
+  var importer = FilesystemImporter(Directory.current.path);
```

Comment:
I'd make this a top-level final variable so you don't have to reinstantiate it over and over—it's stateless so you won't run into problems there. You can also just use `'.'` rather than `Directory.current.path`.

Reply:
Doing that breaks the tests because we override `Directory.current` to be the sandbox for each one. I'll fix this when I fix #29.

Reply:
Hmm, that's arguably a bug in Dart Sass's API... we shouldn't have an instance of `FilesystemImporter` vary its behavior based on the PWD. This code is fine, though.